<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Not Hesaplama Otomasyonu</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Öğrenci Adı:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ders Adı:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vize Notu:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Notu:</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLineEdit" name="lnedit_ad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnedit_dersad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnedit_vizenot">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnedit_finalnot">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buton_hesapla">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buton_temizle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Temizle</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_sonuc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buton_cikis">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buton_temizle</sender>
   <signal>clicked()</signal>
   <receiver>text_sonuc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buton_cikis</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
